Wikipedia/UI-V5/WMFArticlePreviewCell.m

```diff
@@ -6,12 +6,15 @@
 
 #import "NSAttributedString+WMFModify.h"
 #import "UIImageView+MWKImage.h"
+#import "NSAttributedString+WMFHTMLForSite.h"
+#import "UIFont+WMFStyle.h"
 #import "UIButton+WMFButton.h"
 #import "WMFSaveButtonController.h"
 #import "SessionSingleton.h"
+#import "NSParagraphStyle+WMFParagraphStyles.h"
 
-static CGFloat const WMFTextPadding = 8.0;
-static CGFloat const WMFImageHeight = 160;
+CGFloat const WMFArticlePreviewCellTextPadding = 8.0;
```

Comment:
Do you have a reference to demonstrate why removing `static` from file scoped constants is the preferred pattern? 

I ask because this goes against our style guide and we should update it if there is a technical reason to do this: https://www.mediawiki.org/wiki/Wikimedia_Apps/Team/iOS/ObjectiveCStyleGuide#Constants